### PR TITLE
Fix Piwigo startup and docs for #614

### PR DESCRIPTION
**Pull request by darshitpp**
The host name in the docs is incorrect. I managed to make it work as per: https://www.youtube.com/watch?v=hIGibqdSo6c

Also, piwigo refuses to start with the root user. Changed it to `1` for the current user

### CODE REVIEW

File: roles/piwigo/defaults/main.yml [hunk 1]
```diff
@@ -8,8 +8,8 @@ piwigo_data_directory: "{{ docker_home }}/piwigo/data"
 piwigo_photos: "{{ photos_root }}/piwigo"
 
 # uid / gid
-piwigo_user_id: "0"
-piwigo_group_id: "0"
+piwigo_user_id: "1"
```
Comment (davestephens) [inline roles/piwigo/defaults/main.yml:11]: I don't think uid/gid 1 is the right one for this - we should probably use 1000

**Review by davestephens — CHANGES_REQUESTED** [review]
Needs a minor tweak, nearly good to go.

File: roles/piwigo/defaults/main.yml (hunk 1 above)
Comment (darshitpp) [reply roles/piwigo/defaults/main.yml]: Fixed

**Review by davestephens — APPROVED** [review]
LGTM

**Pull request merged**